maquetado/scss/index-grid.scss

```diff
@@ -0,0 +1,207 @@
+  display: grid;
+  grid-row-gap: 130px;
+  grid-template-columns: auto;
+  grid-template-rows: repeat(2, auto);
+  grid-template-areas: 
+    "back"
+    "book-information";
+  margin: auto;
+  max-width: 1000px;
+  width: 100%;
+  padding-top: 40px;
+}
+
+.book-component-grid {
+  grid-area: book-information;
+  display: grid;
+  grid-column-gap: 52px;
+  grid-template-columns: $wolox-logo-image-normal-size-width auto;
+  grid-template-rows: 55px auto;
+  grid-template-areas:
+    "images title"
+    "images book-information-text";
+  width: 100%;
+  padding: 20px;
+}
+
+.images-component-grid {
+  grid-area: images;
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto;
+  grid-template-areas:
+    "images";
+  width: 100%;
+  position: relative;
+}
+
+.back-component-grid {
+  grid-area: back;
+  display: grid;
+  grid-template-columns: min-content auto;
+  grid-template-rows: auto;
+  grid-template-areas:
+    "back .";
+  width: 100%;
+}
+
+.book-text-component-grid {
+  grid-area: book-information-text;
+  margin-top: 45px;
+  width: 100%;
+}
+
+.back-grid {
+  display: flex;
+  font-size: 20px;
+  grid-area: back;
+
+  &:before {
+    content: url(./images/back-arrow.png);
+    margin-right: 15px;
+  }
+}
+
+.badge-grid {
+  position: absolute;
+  right: -25px;
+  top: -15px;
+  transform: rotate(15deg);
+  width: $badge-width;
+}
+
+.title-grid {
+  font-size: 32px;
+  margin-bottom: 40px;
+}
+
+.field-grid {
+  font-size: 19px;
+  margin-bottom: 25px;
+}
+
+.genre-grid {
+  margin-top: 8px;
+}
+
+.title-component-grid {
+  grid-area: title;
+  display: flex;
+  border-bottom: 4px solid $earls-green;
+}
+
+@media (max-width: $breakpoint-desktop) {
+  .index-main-component-grid {
+    grid-row-gap: 20px;
+    max-width: 853px;
+    padding-top: 46px;
+  }
+  
+  .book-component-grid {
+    grid-template-rows: 145px auto;
+    grid-template-areas:
+      ". title"
+      "images book-information-text";
+    padding: 32px 20px 56px 20px;
+  }
+
+  .book-text-component-grid {
+    justify-content: center;
+  }
+  
+  .badge-grid {
+    width: calc(#{$badge-width} * 0.7);
+  }
+  
+  .title-grid {
+    margin-bottom: 0;
+  }
+
+  .title-component-grid {
+    flex-direction: column;
+    border-bottom: 0;
+  }
+
+  .genre-grid {
+    margin-top: 0px;
+    margin-left: 65px;
+  }
+}
+
+@media (max-width: $breakpoint-tablet) {
+  .index-main-component-grid {
+    grid-template-columns: auto;
+    grid-row-gap: 20px;
+    grid-template-areas: 
+      "book-information"
+      "back";
+    max-width: 336px;
+    padding-top: 32px;
+    justify-items: center;
+  }
+
+  .book-component-grid {
+    grid-template-columns: auto;
+    grid-template-rows: repeat(4, auto);
+    grid-template-areas:
+      "title"
+      "images"
+      "genre"
+      "book-information-text";
+    padding: 12px;
+    justify-items: center;
+  }
+
+  .book-text-component-grid {
+    margin-top: 01px;
+  }
+
+  .images-component-grid {
```

Comment:
why not? Its because in mobile screen the wolox image has to be less wide. I can create a class for the image and set the width there but I find this prettier since I already have a class